### DISCUSSION TITLE
How to stop actions in monologue augmentation?

### DISCUSSION THREAD

**Discussion by lifan0127** [microsoft/teams-sdk#1653] [Q&A]
Hi Team, how can we stop the chain-of-thought reasoning in monologue augmentation before its completion?

In a simple example, I have a counter in state (0) and an increment action (+1). If a user mistakenly tell the bot to set the counter at 100, how can they make it stop in the middle?

**Comment by corinagum** [comment]
https://github.com/microsoft/teams-ai/discussions/1578

**Reply by lifan0127** [reply]
Hi @corinagum  I understand that developer can stop an action within `app.ai.action` by returning "STOP". However, my question is about how a user can stop ongoing action execution if they wish to do so. Send a "STOP" message doesn't work. Thanks!

**Reply by corinagum** [reply]
Right now we don't have a way for the user to stop a command.  @govil1811 this may be a good feature.

**Comment by heyitsaamir** [comment]
We're clearing discussions and starting fresh. This discussion has not been acted upon. Please reopen the discussion, or start a new one if this question is still important. Thank you for your understanding.
